client/web/src/components/time/Timestamp.tsx

```diff
@@ -40,8 +40,14 @@ export const Timestamp: React.FunctionComponent<Props> = ({
         }
     }, [date, noAbout, now, strict])
 
+    const tooltip = useMemo(() => {
+        const parsedDate = typeof date === 'string' ? parseISO(date) : new Date(date)
+        const dateHasTime = parsedDate.getHours() !== 0 || parsedDate.getMinutes() !== 0
+        return format(typeof date === 'string' ? parseISO(date) : date, `yyyy-MM-dd${dateHasTime ? ' p' : ''}`)
```

Comment:
Can't we reuse the parsed date here?

```suggestion
        return format(parsedDate, `yyyy-MM-dd${dateHasTime ? ' p' : ''}`)
```

Reply:
At second glance, I wonder if we want seconds here. Otherwise we lose a piece of information here. I think in come use-cases at least that's sort of important. On the code intel auto index view for example I am usually interested in seconds, too so I can find out how long _exactly_ something took.

Reply:
Agree, added seconds (`pp` instead of `p`).